### ISSUE TITLE
up-to-eleven versus playback speed

### ISSUE THREAD

**Issue by grahamperrin** [padenot/up-to-eleven#5]
With (at least) YouTube content: 

* when up-to-eleven is activated, speed is reset to 1

– and speed can be neither accelerated nor decelerated until after the content is reloaded (a second click on the icon of the extension is not a workaround). 

This conflict is most noticeable when, for example, attempting to use the keyboard shortcuts of [YouTube Playback Speed Control](https://addons.mozilla.org/addon/youtube-speed-control/) whilst transcribing content that is quiet. 

Please, can anything be done to work around the issue? 

TIA

**Comment by dregimbal** [comment]
This issue may be related to [bug 1517199](https://bugzilla.mozilla.org/show_bug.cgi?id=1517199) which has recently been closed

**Comment by padenot** [comment]
Yes, we have fixed this in Firefox now, so it should work well. Let me know if this is not the case.
